### PR TITLE
centrally define media resources

### PR DESCRIPTION
**Pull request by giannik**
to avoid scattered file references

### CODE REVIEW

**Comment by jtkech** [comment]
LGTM

File: src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.Edit.cshtml [hunk 1]
```diff
@@ -9,8 +9,8 @@
     var modalMediaId = "Modal" + Html.IdFor(m => m);
     var uniqueName = $"{Model.PartFieldDefinition.PartDefinition.Name}-{Model.PartFieldDefinition.Name}";
 }
-<script asp-src="~/OrchardCore.Media/Scripts/media.js" asp-name="media" at="Foot" depends-on="admin, vuejs, sortable, vuedraggable, jQuery-ui"></script>
-<style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
+<script asp-name="media" at="Foot"></script>
+<style  asp-name="media"></style>
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.Edit.cshtml:13]: why extra space before asp-name?

File: src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField-Attached.Edit.cshtml [hunk 2]
```diff
@@ -14,8 +14,8 @@
     var uniqueName = $"{Model.PartFieldDefinition.PartDefinition.Name}-{Model.PartFieldDefinition.Name}";
 }
 
-<script asp-src="~/OrchardCore.Media/Scripts/media.js" asp-name="media" at="Foot" depends-on="admin, vuejs, sortable, vuedraggable, jQuery-ui"></script>
-<style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
+<script asp-name="media" at="Foot"></script>
+<style  asp-name="media"></style>
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField-Attached.Edit.cshtml:18]: why extra space before `asp`?

File: src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/Index.cshtml [hunk 3]
```diff
@@ -1,5 +1,5 @@
-<script asp-src="~/OrchardCore.Media/Scripts/media.js" asp-name="media" at="Foot" depends-on="admin, vuejs, sortable, vuedraggable, jQuery-ui"></script>
-<style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
+<script asp-name="media" at="Foot"></script>
+<style  asp-name="media"></style>
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/Index.cshtml:2]: why extra space before `asp`?

File: src/OrchardCore.Modules/OrchardCore.Markdown/Views/Media-MarkdownBodyPart.Modal.cshtml [hunk 4]
```diff
@@ -17,8 +17,8 @@
     </div>
 </div>
 
-<script asp-src="~/OrchardCore.Media/Scripts/media.js" asp-name="media" at="Foot" depends-on="admin, vuejs, sortable, vuedraggable, jQuery-ui"></script>
-<style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
+<script asp-name="media" at="Foot"></script>
+<style  asp-name="media"></style>
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Markdown/Views/Media-MarkdownBodyPart.Modal.cshtml:21]: why extra space before `asp`?

File: src/OrchardCore.Modules/OrchardCore.Html/Views/Media-HtmlBodyPart.Modal.cshtml [hunk 5]
```diff
@@ -15,8 +15,8 @@
     </div>
 </div>
 
-<script asp-src="~/OrchardCore.Media/Scripts/media.js" asp-name="media" at="Foot" depends-on="admin, vuejs, sortable, vuedraggable, jQuery-ui"></script>
-<style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
+<script asp-name="media" at="Foot"></script>
+<style  asp-name="media"></style>
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Html/Views/Media-HtmlBodyPart.Modal.cshtml:19]: why extra space before `asp`?

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/Media-HtmlField.Modal.cshtml [hunk 6]
```diff
@@ -15,8 +15,8 @@
     </div>
 </div>
 
-<script asp-src="~/OrchardCore.Media/Scripts/media.js" asp-name="media" at="Foot" depends-on="admin, vuejs, sortable, vuedraggable, jQuery-ui"></script>
-<style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
+<script asp-name="media" at="Foot"></script>
+<style  asp-name="media"></style>
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.ContentFields/Views/Media-HtmlField.Modal.cshtml:19]: why extra space before `asp`?

**Review by MikeAlhayek — APPROVED** [review]

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/Media-HtmlField.Modal.cshtml (hunk 6 above)
Comment (giannik) [reply src/OrchardCore.Modules/OrchardCore.ContentFields/Views/Media-HtmlField.Modal.cshtml]: all fxed

**Comment by MikeAlhayek** [comment]
I think we should also add `at="Foot"` for the styles too.

**Comment by jtkech** [comment]
Yes maybe but looks like it was not the case

**Comment by MikeAlhayek** [comment]
Yes I noticed that we don't use at footer in style. I think because we use footer as default.  I still think we should include it.

@giannik i'll merge this once we add on foot tag. Thanks

**Comment by jtkech** [comment]
From memory for styles the default is `Head`.

Why here styles should be `at="Foot"`?

**Comment by MikeAlhayek** [comment]
I don't think we need to load media styles early on on page load. It can wait until the end of page to load. We load the structure styles like bootstrap in head.

**Comment by jtkech** [comment]
Okay

**Comment by giannik** [comment]
ok added at="Head" explicitly for style

**Pull request merged**